src/web/pages/scanners/dialog.js

```diff
@@ -167,15 +158,9 @@ const ScannerDialog = ({
     value: scannerType,
   }));
 
-  const scanner_credentials = filter(
-    credentials,
-    client_cert_credentials_filter,
-  );
+  const scanner_credentials = filter(credentials);
   const isInUse = isDefined(scanner) && scanner.isInUse();
-  const show_cred_info =
-    isDefined(scanner) &&
-    isDefined(scanner.credential) &&
-    scanner.credential.credential_type === CLIENT_CERTIFICATE_CREDENTIAL_TYPE;
+  const show_cred_info = isDefined(scanner) && isDefined(scanner.credential);
```

Comment:
I think this can be removed as well as the `scanner_credentials` declaration and the conditional `<CertStatus>` element below.
If the ` CertStatus` component is used nowhere else, it can also be removed.